### ISSUE TITLE
Removing a filter is a bit hidden since new design

### ISSUE THREAD

**Issue by yohanboniface** [gristlabs/grist-core#374]
Hi!

Since the new (cool!) new design for managing filter, we have seen users during our training unable to find the way to remove an active filter they just added. They never try to click on the small green hourglass 

![image](https://user-images.githubusercontent.com/146023/208166142-f13fa7f7-fe16-49fb-b03f-fbfdeb56fd02.png)

I'm thinking about two fixes:

- add a small "cross" on hover on the filter itself (was it like this before ? I can't remember now)
- on the popup that opens when clicking on the filter itself, add a trash icon near to the pin icon

Thoughts ? :)

Thanks again!

Yohan

**Comment by yohanboniface** [comment]
Adding a further remark, seen in our user trainings: when they see the new "Save" button, they think it's the way to save the data (while it's only to save the filters).

**Comment by vviers** [comment]
Agreed that the filter design is a bit clunky — maybe worth looking at at some point ? cc @kbizien

**Comment by hexaltation** [comment]
A design is proposed for filters in [#1201](https://github.com/gristlabs/grist-core/issues/1201)
